### PR TITLE
osd/PeeringState: fix acting_set_writeable min_size check

### PR DESCRIPTION
**Pull request by athanatos**

### CODE REVIEW

**Comment by athanatos** [comment]
I think this explains both https://tracker.ceph.com/issues/48613 and the original lrc problem https://tracker.ceph.com/issues/48417

**Comment by athanatos** [comment]
Commit message has an explanation.

**Comment by athanatos** [comment]
@ideepika Can you run this through the teuthology suite sequence you've been using to reproduce the original lrc problem?

**Comment by ideepika** [comment]
> @ideepika Can you run this through the teuthology suite sequence you've been using to reproduce the original lrc problem?

sure!

**Review by gregsfortytwo — APPROVED** [review]
LGTM

File: src/osd/PeeringState.h [hunk 1]
```diff
@@ -2314,7 +2314,7 @@ class PeeringState : public MissingLoc::MappingInfo {
    * applicable stretch cluster constraints.
    */
   bool acting_set_writeable() {
-    return (acting.size() >= pool.info.min_size) &&
+    return (actingset.size() >= pool.info.min_size) &&
```
Comment (tchaikov) [inline src/osd/PeeringState.h:2317]: just a nit, could you put:
```
Fixes: https://tracker.ceph.com/issues/48613
Fixes: https://tracker.ceph.com/issues/48417
```
instead in the commit message?

**Review by ideepika — APPROVED** [review]
passes with reproducer I was using: https://pulpito.ceph.com/ideepika-2021-04-05_06:44:27-rados:thrash-erasure-code-wip-deepika-testing-2021-04-05-0643-distro-basic-gibba/
running with: https://github.com/ceph/ceph/pull/40593 on a full suite run
here: http://pulpito.front.sepia.ceph.com/ideepika-2021-04-05_14:32:13-rados-wip-deepika-testing-2021-04-05-0643-distro-basic-smithi/

**Review by neha-ojha — APPROVED** [review]
this makes a lot of sense

**Comment by tchaikov** [comment]
- https://pulpito.ceph.com/kchai-2021-04-05_10:46:36-rados-wip-kefu-testing-2021-04-05-1650-distro-basic-gibba/
- https://pulpito.ceph.com/kchai-2021-04-06_02:58:54-rados-wip-kefu-testing-2021-04-05-1650-distro-basic-smithi/
- https://pulpito.ceph.com/kchai-2021-04-06_05:17:42-rados-wip-kefu-testing-2021-04-06-1109-distro-basic-smithi/
- https://pulpito.ceph.com/kchai-2021-04-06_06:26:47-rados-wip-kefu-testing-2021-04-06-1117-distro-basic-smithi/

**Pull request merged**